features/user_can_view_sent_message.feature

```diff
@@ -0,0 +1,24 @@
+Feature: User can view sent message
+As a user
+In order to know what messages I have composed
+I want to view a sent page
+
+Background:
+        Given the following user exist:
```

Comment:
Take a look at how we create messages in the `user_can_delete_message` feature file

Reply:
Updated!